minerl/herobraine/env_specs/basalt_specs.py

```diff
@@ -0,0 +1,263 @@
+from typing import List, Optional
+
+import gym
+
+from minerl.env import _fake, _singleagent
+from minerl.herobraine.env_spec import EnvSpec
+from minerl.herobraine.hero import handlers, mc
+
+BUTTON_ACTIONS = {"forward", "back", "left", "right", "jump", "sneak", "sprint", "attack", "use"}
```

Comment:
Brandon: There might be a buttons constant somewhere that we can use instead of redefining this list